### DISCUSSION TITLE
Link desactualizado

### DISCUSSION THREAD

**Discussion by Santiago-Lovelli** [dds-jv/foro#340] [Consultas generales] [answered]
Buenas, estoy navegando por la pagina de [dds](https://dds-jv.github.io/pautas/sobre-la-cursada) y al final el hipervinculo de las clases del final lleva al calendario del 2021 https://dds-jv.github.io/bitacoras/2021/viernes-n/ y no del actual https://dds-jv.github.io/bitacoras/2022/viernes-n/

**Comment by gastonprieto** [comment, marked as answer]
Buenas @Santiago-Lovelli 

Muchas gracias por el aviso, ya lo cambiamos :)
